src/metabase/models/permissions_group.clj

```diff
@@ -100,12 +100,24 @@
     (when group-name
       (check-name-not-already-taken group-name))))
 
+(defn- grant-subscription-permission
+  [group-id]
+  ;; Require via classloader to avoid cyclic dependencies
+  (classloader/require 'metabase.models.permissions)
+  ((resolve 'metabase.models.permissions/grant-permissions!) group-id
+   ((resolve 'metabase.models.permissions/general-perms-path) :subscription)))
+
+(defn- post-insert [group]
+  (u/prog1 group
```

Comment:
Note that even though General Permission is an enterprise feature, we still grant it for the newly created group even for OSS. 
This makes sure everything will work if users upgrade from OSS to EE